pkg/agent/core/ngt/service/kvs/option.go

```diff
@@ -0,0 +1,44 @@
+//
+// Copyright (C) 2019-2021 vdaas.org vald team <vald@vdaas.org>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package kvs
+
+import (
+	"github.com/vdaas/vald/internal/errgroup"
+)
+
+// Option represents the functional option for bidi.
+type Option func(n *bidi)
+
+var defaultOptions = []Option{}
+
+// WithErrGroup returns the option to set the errgroup.
+func WithErrGroup(eg errgroup.Group) Option {
+	return func(b *bidi) {
+		if eg != nil {
+			b.eg = eg
+		}
+	}
+}
+
+// WithConcurrency returns the option to set the concurrency.
+func WithConcurrency(c int) Option {
```

Comment:
Is this used in somewhere? I cannot find anywhere.

Reply:
There was a mistake when fixing the Bug and it disappeared.
https://github.com/vdaas/vald/pull/1376/files#r688324051